src/Main.hs

```diff
@@ -55,10 +56,13 @@ main = do Options f c conf <- execParser opts
           g   <- getRandom
           EConfigWithUsage cfg ks _ <- maybe (pure (EConfigWithUsage defaultConfig mempty mempty)) parseConfig conf
           unless (cfg ^. sConf . quiet) $ mapM_ (hPutStrLn stderr . ("Warning: unused option: " ++) . unpack) ks
+          let cd = corpusDir $ view cConf cfg
+          txs <- loadTxs cd
           cpg <- flip runReaderT cfg $ do
             cs       <- Echidna.Solidity.contracts f
             ads      <- addresses
             (v,w,ts) <- loadSpecified (pack <$> c) cs >>= prepareForTest
             let ads' = AbiAddress . addressWord160 <$> v ^. env . EVM.contracts . to keys
-            ui v w ts (Just $ mkGenDict (dictFreq $ view cConf cfg) (extractConstants cs ++ NE.toList ads ++ ads') [] g (returnTypes cs))
+            ui v w ts (Just $ mkGenDict (dictFreq $ view cConf cfg) (extractConstants cs ++ NE.toList ads ++ ads') [] g (returnTypes cs)) txs
```

Comment:
can we make a ticket to make a default gendict method because this line gets more ass the more we do stuff

Reply:
Sure thing.

Reply:
https://github.com/crytic/echidna/issues/365